packages/extension-chakra-storefront/src/pages/checkout/partials/payment.jsx

```diff
@@ -29,18 +29,9 @@ const Payment = () => {
     const {formatMessage} = useIntl()
     const {data: basket} = useCurrentBasket()
     const selectedShippingAddress = basket?.shipments && basket?.shipments[0]?.shippingAddress
-    //TODO: Change to const after deleting the test/mocked data
-    let selectedBillingAddress = basket?.billingAddress
-    let appliedPayment = basket?.paymentInstruments && basket?.paymentInstruments[0]
+    const selectedBillingAddress = basket?.billingAddress
```

Comment:
Just removed mocked data that was added for Payment component testing. There are no other changes in this file